.env.example

```diff
@@ -8,6 +8,10 @@ PORT=5000
 
 DEBUG=false
 
+# logging into files inside LNBITS_DATA_FOLDER
+LOG_FILE=true
+LOG_RETENTION="30 days"
```

Comment:
- let make it 90 days, storage is cheap

Reply:
```suggestion
LOG_RETENTION_DAYS=30
```
- users might have a typo, use upper case, miss adding a space (`30days`), etc. Better to just have a number value.

Reply:
hm, i feel like keeping it i string is more verbose, you could do `1 week`, or `3 days`.

https://loguru.readthedocs.io/en/stable/overview.html#easier-file-logging-with-rotation-retention-compression

Reply:
- ok, I see. Please add some examples in the comment then.